src/ir/predicate.h

```diff
@@ -111,16 +113,17 @@ class Implies : public Predicate {
 
   virtual ~Implies() { }
 
-  std::string ToDatalogRuleBody(const AccessPath &ap) const override {
+  std::string ToDatalogRuleBody(
+      const AccessPath &ap, const DatalogPrintContext &ctxt) const override {
     // To make it so that Souffle has first class information that our
     // antecedent is the same in both places it is used, we assign it to a
     // variable within the rule.
-    std::string antecedent_str = antecedent_->ToDatalogRuleBody(ap);
+    std::string antecedent_str = antecedent_->ToDatalogRuleBody(ap, ctxt);
```

Comment:
Ugh, the comment above this line is wrong and it is my fault. I initially wanted to assign the `antecedent_str` to some kind of temporary within the rule, but that isn't possible (or I couldn't figure it out). Could you remove the above comment? I can fix it later if you do not want to clean up my mess, but since you have altered this file already, thought I'd ask.

Reply:
Done in https://github.com/google-research/raksha/pull/230. (I generally try to keep PRs single purpose. It makes it easier to cherry-pick, roll back, etc.)

Reply:
Thanks.